open-metadata-implementation/adapters/open-connectors/event-bus-connectors/open-metadata-topic-connectors/kafka-open-metadata-topic-connector/src/main/java/org/odpi/openmetadata/adapters/eventbus/topic/kafka/KafkaOpenMetadataEventConsumer.java

```diff
@@ -142,7 +141,7 @@ public void run()
                 
             	//if we are close to the timeout, force a poll to avoid having the consumer
             	//be marked as dead because we have not polled often enough
-            	boolean pollRequired = System.currentTimeMillis() > maxNextPollTimestampToAvoidConsumerTimeout;
+            	final boolean pollRequired = System.currentTimeMillis() > maxNextPollTimestampToAvoidConsumerTimeout;

```

Comment:
If the intention of this change is to modernise the code, then should these calls to currentTimeMillis() not be replaced with the more modern time classes, especially as duration is used below

Reply:
In this case it has no meaning)